tests/evm_runtime_tests.cpp

```diff
@@ -1101,12 +1101,14 @@ struct evm_runtime_tester : eosio_system_tester, silkworm::State {
 };
 
 BOOST_AUTO_TEST_SUITE(evm_runtime_tests)
-BOOST_FIXTURE_TEST_CASE( GeneralStateTests, evm_runtime_tester ) try {
+BOOST_AUTO_TEST_CASE( GeneralStateTests ) try {
```

Comment:
I thought pretty hard about how I could avoid removing the fixture (and thus needing to plumb through all the `t.` below, among the other refactoring). Just couldn't come up with anything. The problem is the only ctor of `validating_tester` that allows either a custom config or modifying the config (what is used in current approach), also requires an externally created tmpdir to ensure the tmpdir outlives the tester. It's too bad there isn't a variant of the ctor that _just_ has the config modification callback so that `validating_tester`'s internal tmpdir can continue to be used.